src/app/researcher/study/request/[memberIdentifier]/form.tsx

```diff
@@ -39,39 +44,173 @@ export const Form: React.FC<{ memberId: string; memberIdentifier: string }> = ({
+                                <Accordion.Panel>
+                                    <Divider my="sm" mt={2} mb={5} />
+                                    <Text w="90%">
+                                        This section is here to help you submit your study proposal. Consider providing
+                                        as much detail as possible to ensure the Reviewer has all the information needed
+                                        to make an informed decision.
+                                    </Text>
+                                    <Group mt={30}>
+                                        <Stack>
+                                            <Flex direction="column" gap="xl">
+                                                <Text>Study Title</Text>
+                                                <Text>Study Lead</Text>
+                                                <Text>Principal Investigator</Text>
+                                                <Text>Study Description</Text>
+                                                <Text>IRB Document</Text>
+                                                <Text>Agreement Document</Text>
+                                            </Flex>
+                                        </Stack>
+                                        <Stack mt={10}>
+                                            <Flex direction="column" gap="sm">
+                                                <TextInput
+                                                    placeholder="Enter a title (max. 50 characters)"
+                                                    w={536}
+                                                    bd="1px solid #000000"
+                                                    required
+                                                    name="title"
+                                                    key={form.key('title')}
+                                                    {...form.getInputProps('title')}
+                                                    className={customLabel}
+                                                />
+
+                                                <TextInput
+                                                    w={536}
+                                                    bg="#ddd"
+                                                    bd="1px solid #000000"
+                                                    disabled
+                                                    name="title"
+                                                    // value={user?.emailAddresses[0].emailAddress}
+                                                    value={user?.firstName + ' ' + user?.lastName}
+                                                    data-testid="study-lead"
+                                                />
+
+                                                <TextInput
+                                                    w={536}
```

Comment:
ideally we won't have to use fixed widths on any of this stuff, can we just rely on flex/layout components and let them be responsive? if its totally required to have a fixed width, we could just set the parent component's width 🤔 but hopefully that won't be necessary either 😄